hollow/src/main/java/com/netflix/hollow/core/index/HollowHashIndex.java

```diff
@@ -163,6 +163,10 @@ private boolean matchIsEqual(FixedLengthElementArray matchHashTable, long hashBu
                     HollowObjectTypeReadState objectAccess = (HollowObjectTypeReadState)readState;
                     readState = objectAccess.getSchema().getReferencedTypeState(fieldPath[j]);
                     hashOrdinal = objectAccess.readOrdinal(hashOrdinal, fieldPath[j]);
+                    // Cannot find nested ordinal for null parent
+                    if(hashOrdinal == -1) {
```

Comment:
Minor note: can we update these `-1`s to be `HollowConstants.ORDINAL_NONE`?

Reply:
Sure, done.